marimo/_server/session/session_view.py

```diff
@@ -98,6 +101,13 @@ def add_operation(self, operation: MessageOperation) -> None:
             self.cell_operations[operation.cell_id] = merge_cell_operation(
                 previous, operation
             )
+            if not previous:
+                return
+            if previous.status == "queued" and operation.status == "running":
+                self.set_execution_time(operation.cell_id, "start")
+            if previous.status == "running" and operation.status == "idle":
```

Comment:
I think it can be interrupted to also end the time (not sure what we actually show on the frontend today if interrupted)

Reply:
When cell execution is interrupted, the time elapsed before interruption is stickied to the cell. With these changes, the behavior is the same after session resume:

<img width="1174" alt="Screenshot 2024-06-13 at 7 50 00 PM" src="https://github.com/marimo-team/marimo/assets/7220175/620c03a9-14cf-48ee-9538-2a14ebe74025">